client/verta/verta/_internal_utils/_utils.py

```diff
@@ -155,6 +158,29 @@ def _request_to_curl(request):
 
         print(curl)
 
+    def get_workspace_name_from_legacy_id(self, workspace_id):
```

Comment:
I _think_ this might not be necessary anymore as things now return the new id too. Need to double check and not a blocker

Reply:
I'm erring on the side of keeping it, in the interest of "the logic ain't broke", but could be revisited and cleaned up in the future